### ISSUE TITLE
Suggestion for better server

### ISSUE THREAD

**Issue by Chungers01** [SmashMC-Development/Suggestions#839]
**Describe the suggestion**
The server is obviously still great, but it is just not fun to play on as a HA hunter/breeder.

I played basically since when we had lucario, eevee, and ludicolo servers, and in my opinion, server was more fun before this last reset by far.  Very few players on rayquaza server tried to keep the eco alive and make battle perfect (bp) and bp hidden ability pokemons actually worth something.  

However, with the maximum cap in gts on rayquaza server, which are 20k for bp and 40k for bpha, any good mons are limited to sell for 40k as people say "oh 400k for bpha grookey is too expensive" which is completely not true as it was sold for around 1mil when people first sold them before the last reset.  On rayquaza, froakie, litten, galarian slowpoke, torchic, etc are already sold for 40k.  Economy is already killed.  I know more than 5 people who I've talked to on the server that left because of this.

And also the fact that rare pokemons such as grookey, snivy, sobble, etc spawn like wayyy too much in warzone also decreases the value of those pokemons when it comes to the economy.  The more the rare pokes spawn in wz, the higher chance of getting its HA.  This obviously causes more people to have breedable HAs that are SUPPOSED to be rare, but just gets it so easily only because they camp wz for it instead of actually hunting for them in the wild.  For me personally and few of my friends, this is one of the main reason we are losing interest in playing on this server.  

These problems can be fixed very easily in my opinion as my idea should be fair for everyone:

1. Create a minimum price for pokemons but remove any maximum prices for them on gts  -->  This prevents everyone from underselling any pokes while allowing them to come up with their own prices for them.
2. If anyone is caught underselling (selling below the amount set as minimum price) should be banned or not be able to sell any pokes in the future.
3. Fix the spawning in warzone so that it doesn't spawn too many of rare pokemons like grookey, sobble as their value is dropping rapidly as people are catching its HA in wz and eventually kills the eco for them
4. Make pokemons from warzone unbreedable when they leave warzone  -->  This will definitely keep eco alive and more people who actually hunts pokemon will actually be able to enjoy the game.

I'd be happy to gather good players on rayquaza server and decide on the minimum prices on mons

Please help sort this situation out. Thanks.

**Which servers does this apply to?**
Rayquaza
Probably other servers too

**Additional context**
N/A

**Comment by DinglePringleUwU** [comment]
> Fix the spawning in warzone so that it doesn't spawn too many of rare pokemons like grookey, sobble as their value is dropping rapidly as people are catching its HA in wz and eventually kills the eco for them

This is all so true. We do have a similar problem on Kyogre - tho it is not nearly as bad as on rayquaza.

**Comment by Chungers01** [comment]
I really hope devs see this and if they care about the server, they should fix this immediately.  This kills the fun of playing on the server in my opinion.

**Comment by ThePapaChunks** [comment]
90% of markets are crashed on Groudon because everyone is just selling bp breedables at 20k while the price of bp unbreeds was 80k before the glitch.

**Comment by ghost** [comment]
I agree with most of the points...but implementing things like reducing rare wz spawns wouldn't help now because most of the players already have good HAs and it will only discourage newer players because they wouldn't be able to do the same easily. In my opinion, easy access to HAs is not really a bad thing as the server is battle centric. But, prices definitely needs to be monitored in a much better manner.

**Comment by Chungers01** [comment]
I understand that newer people would have less access to HAs if wz gets renewed, however, the more people have the HA of pokes like grookey, mareanie, etc, their value will drop rapidly and eventually cause them to sell for 20k 40k whatever the amount people are selling for right now.  This is bad bc some people would try to save 500k for bpha grookey unbreed while some people just camp wz until they find one with ubers and try to kick everyone else out of it so that they can check all grookeys by themselves.  Harder access to a pokemon means some people are willing to pay the price so that they can have the advantage over the opponent.  This is why I'm saying wz and gts needs to be renewed so that we can at least keep the current eco somewhat alive.

**Comment by ghost** [comment]
You can't expect to get premium for HAs like grookey, toxa etc forever. Only the first few ppl to catch these rare HAs get those very high prices. Prices for them fall down to normal HA process eventually. Don't get me wrong, there is a need for better base price for HAs but prices like 500k are premium which only first few ppl to find them get.

**Comment by Mikirae** [comment]
So, the maximum on GTS is actually a bug, and we're trying to figure out if it's a config option or a bug in the plugin itself, so please be patient with us there :)
I can go through and make the rarities of certain mons in Warzone less than those of others like we had in the past Warzone. Mons that come to mind are Starters, OU mons, Ditto, and Eevee. Any others that I'm not thinking of?
Might not help too much in the short run, but I think it would have success in the long run.
Making mons unbreedable upon leaving Warzone was something that I had thought of initially, but I figured people would be very mad about it so I believe we chose not to implement it.

**Comment by Chungers01** [comment]
Good to hear GTS is being worked on :)
Warzone wise, starters spawn way too much so decreasing the rate would definitely be good and ditto is easily accessible from raids so I don't think we need that.  Pokemons that we have to, for example, fish for such as mareanie spawns a lot, but dratini never spawns.  So balancing each pokemon rates would be great, but obviously, there's too many lmao.  
And if the spawn rate is being fixed, then I guess making mons unbreedable when leaving warzone might be unnecessary.  If the spawn rates get balanced, then all should be good.

Also, I noticed some legendaries like diancie, tapus (koko, lele, bulu, fini), lake trios (uxie, mesprit, azelf), landorus, tornadus, thundurus, etc don't spawn in warzone.  If possible, balancing the spawn rates of all legendaries to equal would make warzone be better.

**Comment by ghost** [comment]
So...I strongly dislike the idea of lowering rates of starters, eevee and OU pokemon in wz. As a person who has been in warzone literally everyday for over 1 hour....I still haven't got any HA starter other than HA litten....so I don't really think it's broken in any way. Plus with the timer and wz fees, I don't think a person can spend a lot of time in wz each day anyway.

**Comment by RatedPG21** [comment]
I have spent ALOT of time in warzone, with only one ha starter, but I do agree that they should be lowered and or be made unbreedable when you leave warzone, I like the idea of keeping some Pokémon rare so that there is joy when you do find them or there ha.

**Comment by Mikirae** [comment]
Well it's what we did before. Except I think I specifically did starters,
Eevee, Ditto, Pex, and whatever else someone asked me to do. Lol. I mean I
don't mind either way. I'll come back to this thread in a couple of days
and see if there's been any good discussions on it, as I honestly see both
sides here.
This is one of those things that people will probably be angry about either
way, sadly. No simple solution here to solve everyone's qualms.

**Comment by ghost** [comment]
In my opinion...it's not broken in any way...just check how many servers have HA alolan ninetale after 2 months of reset....and they spawn a lot in wz....it was only caught in wz yesterday on Groudon.....plus most of the HA grookey were found outside warzone on Groudon....HA grookey were found inside wz too but after ppl had spend like a month in wz....so I don't see it as broken...like I said I only have found 1 HA starter in wz so it's definitely not overpowered in any way. Even eevee and eeveelution spawn a lot in wz....but only 1 or 2 HA eevee are found in wz yet. Moreover, the ppl who find HAs in wz spend a lot of time finding them...usually over a week or even more....which would be the same time you would spend overworld to catch an HA. The only reason wz spawns appear to be overpowered are because there is more variety there compared to overworld but one can argue that is the benefit of taking the risk to go wz. I mean you do have to reward the risk player is taking going to wz.

**Comment by Chungers01** [comment]
Like u just said, people find HAs outside of warzone a lot.  This is also a reason why we don’t need there to be 20 grookeys spawning in warzone for just one person that runs ubers all the time and trying to kick others who tries to check those grookeys to see if they are HAs.  Warzone is not a place to hunt HAs.  Its where we are supposed to fight others and catch rare legendaries that don’t spawn often in overworld.  I personally found more than 40 HAs outside of warzone and a lot were from raids.  Raids are already overpowered but this is fine bc it is available for everyone.  Wz is not.  There’s always going to be one person just sitting in there waiting for grookey to spawn.  To me, this doesn’t seem fair to newbies.  There is absolutely no reason for warzone to have that many rare starters like torchic, snivy, sobble, grookey, etc to spawn.

Now look.  Before reset, every server had HA grookey, scorbunny, snivy, torchic, etc.  I have never seen them spawn in warzone before this reset lmao.  This just shows that we don’t need these “supposed to be rare” pokemons to spawn in warzone way too much for basically a one friend group to sit in warzone and share them within the group.  I found 3 HA grookeys in warzone in last 4 days and I purposefully killed them so no one else can get it and ruin the market.  Just so you know, I made this gothub before I got my HA grookey.  While I could hunt for more HAs in warzone and abuse the system, I prefer to have more equal opportunities for everyone and actually let people enjoy themselves when they finally get the HA poke that they’ve been hunting for.

Like I said earlier, there is absolutely no reason for rarer pokemons such as starters to be spawning like they are right now in warzone.

**Comment by RatedPG21** [comment]
> Like u just said, people find HAs outside of warzone a lot. This is also a reason why we don’t need there to be 20 grookeys spawning in warzone for just one person that runs ubers all the time and trying to kick others who tries to check those grookeys to see if they are HAs. Warzone is not a place to hunt HAs. Its where we are supposed to fight others and catch rare legendaries that don’t spawn often in overworld. I personally found more than 40 HAs outside of warzone and a lot were from raids. Raids are already overpowered but this is fine bc it is available for everyone. Wz is not. There’s always going to be one person just sitting in there waiting for grookey to spawn. To me, this doesn’t seem fair to newbies. There is absolutely no reason for warzone to have that many rare starters like torchic, snivy, sobble, grookey, etc to spawn.
> 
> Now look. Before reset, every server had HA grookey, scorbunny, snivy, torchic, etc. I have never seen them spawn in warzone before this reset lmao. This just shows that we don’t need these “supposed to be rare” pokemons to spawn in warzone way too much for basically a one friend group to sit in warzone and share them within the group. I found 3 HA grookeys in warzone in last 4 days and I purposefully killed them so no one else can get it and ruin the market. Just so you know, I made this gothub before I got my HA grookey. While I could hunt for more HAs in warzone and abuse the system, I prefer to have more equal opportunities for everyone and actually let people enjoy themselves when they finally get the HA poke that they’ve been hunting for.
> 
> Like I said earlier, there is absolutely no reason for rarer pokemons such as starters to be spawning like they are right now in warzone.

I agree with this, warzone is a place to fight and catch legendaries, that’s the whole point of it, having people come In with throw teams just to ha hunt starts to get annoying. I personally think making Pokémon such as starters and such rarer is a great idea to make wz much more competitive.

**Comment by ghost** [comment]
Everyone had rare HAs before reset because you were allowed to breed HAs if you were not the OT. In an economy where you are just allowed to breed the HAs you catch, you definitely need them to spawn in warzone. Warzone is definitely the place to battle but pokemons that spawn there are reward for taking that risk. People like to complain about ppl camping wz with ubers....what's stopping you from doing the same? As for whoever is hunting HA in wz with throws...they are doing it with a risk of getting their HA stolen...you do know you become the OT when you steal HA caught in wz right? I personally have stolen HAs in warzone from ppl's bank. As for people who are their with throws...just let them catch a legend and then steal it, I guess.
Like I mentioned in another thread...in my opinion ppl who don't want to battle will always bring throws to wz....no matter how easy you make for them to get BPs.

**Comment by Chungers01** [comment]
I never complained about people sitting in warzone with ubers lmao. I’m personally fine with whatever people do.  But, here’s the thing.  You also mentioned most people find HAs outside of warzone.  Those most people definitely spent a lot of time to hunt them, making platform and waiting for them to spawn.  This obviously shows that hunting in the wild is doable and warzone don’t need to be fed up.

Also, you can’t just say spawn rate in warzone is fair because you only got 1 or 2 of them.  To me, that just sounded like you wanted to just keep your easier way of finding rare HAs.  

From my experiences from before resets (ray and lucario server) I’ve hunted all my HA pokes in the wild and never had a problem with it. On lucario, my friend and I had roughly about 10~11 full boxes of different HAs all caught in the wild between just 2 of us.  I’ve had hunts that went on for 2 weeks.  There were no spawning boosts back then and still, there was no problem in hunting.  

I get that people want easier ways to find rarer things, however, that is how the value of things decrease as people stop appreciate the fact that they have something that is very rare.  Easier access to rare HAs kills that eco 100%.  This is why my friend and I came up with the idea that only OT should be able to breed HAs, OT basically meaning people who actually put the work in to hunt for those pokes.  

While I understand why some people want to keep the same spawn rates, however, the reasoning behind why they think it should remain the same seems very invalid to me as it does not do any good for the server and basically benefits the only select few.

**Comment by ghost** [comment]
> This is also a reason why we don’t need there to be 20 grookeys spawning in warzone for just one person that runs ubers all the time and trying to kick others who tries to check those grookeys to see if they are HAs.

First of all, to me, it feels like you are complaining about warzone and people with Ubers (which I am not one of btw) just because it makes it harder for you to go warzone and hunt HAs. The only reason you are complaining about warzone spawns is because you have difficult time accessing warzone. Honestly, if you were worried about HAs, you would point to raids first. Raids is how people are getting the HAs the most, not warzone. Warzone spawns are time based, so you don't even get rare pokemon like scorbunny, dratini and gible in warzone which are abundantly present in raids.

That being said, I was pointing out that I only got 1 HA starter from warzone because that is the proof that warzone is not too overpowered and it's not just me as you can see the quote below from RatedPG. I know for a fact that RatedPG has spent almost the same amount of time in warzone as I have. He might not agree with me here but his reason is different.

> I have spent ALOT of time in warzone, with only one ha starter, but I do agree that they should be lowered and or be made unbreedable when you leave warzone, I like the idea of keeping some Pokémon rare so that there is joy when you do find them or there ha.

When you spend the amount of time I do in warzone, hunting HAs and otherwise, you will realize it is not overpowered. I have been on warzone leaderboard from the beginning of the reset so I can say that I have spend a lot of time in there.

> I get that people want easier ways to find rarer things, however, that is how the value of things decrease as people stop appreciate the fact that they have something that is very rare. Easier access to rare HAs kills that eco 100%. 

The majority of HAs most people have caught are from overworld and raids...So, I don't know why you are complaining about warzone spawns....they are perfectly fine. You are assuming that it is easier to catch HAs in warzone....it is not. It almost takes the  same amount of time as overworld and raids. You not only spend the same amount of time looking for them....you also spend a lot of money for warzone timer to look for them. Then, there is a added risk of losing your pokemon or the HA you caught in warzone. I personally have spent over 3mil+ on warzone and got 1 HA starter out of it....you don't think it is fair?

> While I understand why some people want to keep the same spawn rates, however, the reasoning behind why they think it should remain the same seems very invalid to me as it does not do any good for the server and basically benefits the only select few.

It is suppose to benefit a select few. That's the whole point of high risk, high reward.

Hope this makes more sense.

**Comment by Chungers01** [comment]
First of all, I’ve beaten several ubers so I don’t need to complain about that. I personally have no problem with people abusing with ubers in warzone.  If I was to complain about it, I would’ve made a suggestion for separating warzone: one for ubers and one for OU and below.

I don't think u understood the main reason for this suggestion.

Making rare pokemons with good HAs easily accessible increases the chances of finding HA of it significantly easier than what it should be.  The reason why they sell for a lot is because they are "rare".  

Ray has many people that don’t care about eco and easily gets their HAs in warzone to end up selling bpha for 40k on gts. And I have friends on other servers who are experiencing the same issue.  I’ve talked to the undersellers on ray and tried to get them to not sell on gts and actually give value to the pokemon, but they said they don’t care what happens to eco as long as they get money. This includes litten, snivy, gible, mareanie, etc. partially fault of gts glitch, however people who don’t care about eco gets their hand on HA and they just completely destroy it.

And you also mentioned it was easier to get HAs because you didn’t have to be the OT of the poke to breed before this reset. Look at what happened to them. Eco murdered. It is obvious that easier access to HA pokes kills the economy. That is why we came up with the idea that only OT can breed HAs. What is the point of having only OT be able to breed when the rare pokes are very easy to hunt in warzone? To me, this is basically the same thing as being able to sell breedable HAs because they both essentially have the same impact on the economy.

Also you said you spent most amount of time and ended up with just HA litten and few others while other people got more HA starters than you did.  This is obviously the main reason why you want to keep the same spawn rates.  You obviously don't care about what impact this could have on the server's economy.  If we keep the same spawn rates, then more people will be getting the rare HA pokes easily, and by the time you get your hands on one, market for that will be destroyed.  It is 100% easier to hunt HAs in warzone because I know a person who got around 40+ different HAs in warzone in few weeks.  I personally got several HAs in warzone even though I go in warzone every once in a while. 

Also, you keep saying you only got 1 HA starter in warzone after all the time you spent in warzone.  I know for fact that you got other rare HAs such as mareanie, gligar, etc.  Starters aren't the only pokemons that are rare and have good HAs lmfao.  You obviously didn't mention other HAs you got in warzone so that you can say "I only got HA litten out of all the starter in warzone after spending 3 million".

Also you can get unlimited amount of time in warzone if you just keep killing pokemons so money spent can be easily be recovered by selling all the legendaries you got from warzone.  So don't be saying the amount of money you spent to hunt HAs in warzone makes it unfair.  If you spent the most time in warzone then you should know money is very irrelevant for staying in warzone.

You also mentioned that it makes it easier for newer players to catch up. This isn’t a game that people need to catch up on others to be on the same page lmao. If they choose to hunt HAs, great for them. If they decide to buy unbreed bpha, great for them. Either way they can get the things that they need without a problem. What matters is putting in the time to actually gather the things you need to enjoy the server. All my friends who were HA hunters left the server because of random people selling rare HA pokes for very cheap and this is why now only the OT can breed HA pokes.

The main problem with this is that it kills the economy very fast.  We only allow OTs to breed HAs for a reason.  Prevent people who don't put the effort and don't care about economy from getting their hands on rare pokemons and keep the eco alive. 

You have yet to give me any good reasoning behind why you think it is okay to keep the same spawn rates.  Give me an actual valid reason instead of giving me "I only got 1 HA starter from spending 3mil on warzone" as this is a very personal problem you have there.  This suggestion is for a better server and it is not to help anyone get rare HAs easily in warzone.

**Comment by ghost** [comment]
I don't want to argue. I have made my point. Warzone is not broken in terms of HA rates. I have only found HA eevee and HA litten in warzone that too after weeks of time in there . If you don't think that's fair rate then I don't know what to tell you. You think it is easier but it really isn't. If anything is broken, it is the raids.  But, I don't see you complaining about those. Moreover, however rare the HA maybe, it eventually drops to normal prices. Usually doesn't take more than a month. You can not expect to get premium for rare HAs forever. People will get there hands on HAs and most of them get it in raids. I don't know if this is true for other servers but on Groudon 70% of HAs on the server are from raids, 20% from hunting and only 10% from warzone. You can ask around on Groudon to confirm. We on Groudon have actually managed to preserve our economy because we create tickets whenever someone undersells on GTS. Whatever the staff think is best.

PS : Please don't assume stuff about me. Saw a couple of personal jabs there :). Also if you really have problem about the economy on the server, I would suggest reporting people who are underselling instead of talking to them yourself.
